pulsar-testclient/src/test/java/org/apache/pulsar/testclient/PerfClientUtilsTest.java

```diff
@@ -77,7 +78,7 @@ public void testClientCreation() throws Exception {
         final ClientConfigurationData conf = builder.getClientConfigurationData();
 
         Assert.assertTrue(conf.isTlsHostnameVerificationEnable());
-        Assert.assertEquals(conf.getAuthPluginClassName(), MyAuth.class.getName());
+        Assert.assertEquals(conf.getAuthentication().getClass(), MyAuth.class);
```

Comment:
Why change this line?

Reply:
That's my bad, I wanted to make that change on the adminCreation test case, because the PulsarAdminBuilder doesn't store the authPluginClassName in the ClientConfigurationData object, rather just gets the instance of the Authentication and keeps that. I must have messed up the line numbers for the change.